### PR TITLE
use beta container image

### PR DESCRIPTION
**Pull request by kain88-de**
I'm trying the new image checking if we face problems. 

For more information: 

https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2

### CODE REVIEW

**Comment by kain88-de** [comment]
The new image seems to give a huge speed boost to our tests. I'm not sure what the reason for that is.

**Comment by orbeckst** [comment]
_Maybe_ the new containers run on better hardware or there is less contention for resources, i.e., we actually do get the equivalent of two cores.

Unfortunately, coverage times out again but everything passes.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 language: generic
 
+sudo: false
```
Comment (orbeckst) [inline .travis.yml]: According to https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2 this should be
```
sudo: required
```
and not `false`.

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+dist: trusty
+
```
Comment (orbeckst) [inline .travis.yml]: According to the blog, you need to enable the line
```
group: edge  # Add this
```
to get the bleeding edge.

Comment (orbeckst) [inline .travis.yml]: *However*, when I look at the Travis report https://travis-ci.org/MDAnalysis/mdanalysis/jobs/244601325 it says explicitly _ This job ran on our container-based trusty beta. Please read our blog post about the public beta._ so the blog instructions are not very clear.

File: .travis.yml (hunk 1 above)
Comment (orbeckst) [inline .travis.yml]: *However*, when I look at the Travis report https://travis-ci.org/MDAnalysis/mdanalysis/jobs/244601325 it says explicitly _This job ran on our container-based trusty beta. Please read our [blog post about the public beta](http://blog.travis-ci.com/2016-11-08-trusty-container-public-beta/?utm_source=trusty-container-based-beta-notice&utm_medium=banner&utm_campaign=trusty-container-based-beta)._ so the blog instructions are not very clear.

**Comment by orbeckst** [comment]
I will restart the "timed out tests" 
- minimal
- full
and see if they complete with coverage.

**Comment by kain88-de** [comment]
I would just wait for the normal switch to happen tomorrow no need to make specific changes now. I just wanted to make sure that there are no interferences with our build.

**Pull request closed without merging**